.github/workflows/build-and-push-containers.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and push containers
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release-container:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Compile tag list
+        id: tags
+        run: |
+          TAG=${GITHUB_REF/refs\/tags\//}
+          PREFIX=ghcr.io/laminas/automatic-releases
+          MAJOR="${PREFIX}:$(echo ${TAG} | cut -d. -f1)"
+          MINOR="${MAJOR}.$(echo ${TAG} | cut -d. -f2)"
+          PATCH="${PREFIX}:${TAG}"
+          echo "::set-output name=tags::[\"${MAJOR}\",\"${MINOR}\",\"${PATCH}\"]"
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.CONTAINER_USERNAME }}
```

Comment:
These secrets should be prefixed, to make it clear they are local to our build

Reply:
Done; used `AUTOMATIC_RELEASES_` as the prefix.